src/frontend/screens/SignInScreen/SignInScreen.js

```diff
@@ -1,8 +1,71 @@
-import React from "react";
+import React, { useState } from "react";
+import "./SignInScreen.css";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../contexts";
+import { signInService } from "../../services"
 
 function SignInScreen() {
+    const [user, setUser] = useState({
+        email: "",
+        password: "",
+    });
+
+    const [passwordToggle, setPasswordToggle] = useState(false);
+
+    const { setAuth } = useAuth();
+    const navigate = useNavigate();
+
+    const signInHandler = async (user) => {
+        const authToken = await signInService(user);
+        if (authToken !== undefined) {
+            localStorage.setItem("AUTH_TOKEN", authToken);
+            setAuth((auth) => ({
+                ...auth,
+                status: true,
+                token: authToken,
+            }));
+            navigate("/");
+        }
+    };
+
     return (
-        <div>SignInScreen</div>
+        <div className="login__container flex--column">
+            <form className=" login__form flex--column" onSubmit={(e) => {
+                e.preventDefault();
+                signInHandler(user, setAuth, navigate);
```

Comment:
your `signInHandler` just takes `user` as a parameter. `setAuth` & `navigate` parameters aren't required.

Reply:
yup would look into it